program/find-sum-of-squares-of-numbers-by-recursion/FindSumOfSquaresOfNumbersByRecursion.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+class sumOfSquares {
+  static int Sum(int n) { 
+    if(n == 1)
+      return 1;
+    else
+      return (n*n + Sum(n-1));
+  } 
+
+  static void Main(string[] args) {
+    Console.WriteLine("Sum of squares of first 5 natural numbers: " + Sum(5));
```

Comment:
Can this be updated to calculate sum of first n natural numbers instead of 
 first 5  natural numbers as the question is not specific to first 5 numbers? 
 https://github.com/codinasion/program/issues/483
  just a heads up :) 
  @anandfresh

Reply:
Can be contributed :)